main/getting-started/intro-zoe.md

```diff
@@ -0,0 +1,279 @@
+
+
+## Satisfying and completing offers
+
+
+At this point, both the offers that Alice and Bob made are known to the contract instance. The Atomic Swap contract code determines that they are matching offers.  The contract instance calls `reallocate`, which *reallocates* the amounts which are the accounting records within Zoe. Payouts are not created here. 
+
+
+The contract instance then *completes* the offers. A call to `zcf.complete()` with both offers as arguments, makes the payouts to the offer holders.  This takes the amounts from the account records, and withdraws the amounts specified by the offers from the digital assets escrowed within Zoe. It's only at the `complete` step that *amounts* are turned into real payments. This is when the payout *promise* resolves into payments. 
+
+
+The offers are now completed, and nothing more is or can be done with them. Zoe deletes completed offers from the contract instance.
+
+
+## Auction example
+
+
+Let’s look at a more complicated example: an auction, where many users might make a bid offer for something, but only one offer will win the auction. Assuming everything validated, Zoe would give the holder of the auctioned item the escrowed amount from the winning offer, and give the holder of the winning offer what they wanted, the item that had been up for bid.
+
+
+But what about all those other bidders who escrowed their bid amounts with Zoe and didn’t win? Zoe guarantees they all get a *refund* of their escrowed amount. Zoe’s *offer safety* guarantees offer makers either get what they wanted or get refunded whatever they put in Zoe’s escrow with their offer.
+
+
+It’s even possible to get both what you wanted in an offer, and at least a partial refund of what you offered. Consider an auction where you make a bid offer where you’re willing to pay up to 10 quatloos, but if you can win with a lower bid, that’s your offer. You win the auction, but your winning bid amount is only 8 quatloos. The payout would resolve to both the item up for auction you get as the winning bidder and the refund of the 2 extra quatloos you escrowed in the offer. 
+
+
+## Other things to know about Zoe
+
+
+A *Dapp (distributed application)* is a combination of a Zoe contract and a server that’s running the back and front ends, and a front end that may be connected to the user’s wallet.
+
+
+Zoe has two distinct APIs. One is the widely accessible, singular Zoe service object that smart contract clients use to provide offers and otherwise interface with contracts. The Zoe service object is a remote object, and so should be accessed with `E(zoe).<api_method>`, which returns a promise. For more on E() see [here](https://agoric.com/documentation/distributed-programming.html#communicating-with-remote-objects-via-e). 
+
+
+The other is the internal API object, referred to as `zcf` that each contract instance uses to interact with Zoe about that contract’s own offers and state (e.g., to do `reallocate` and `complete`).
```

Comment:
"The other is the API for Zoe's Contract Facet, referred to as `zcf`, that each contract instance uses to"